src/test/java/HWS/HW15/BstackdemoTest.java

```diff
@@ -0,0 +1,63 @@
+package HWS.HW15;
+
+import HWS.TestHelper.BaseTest;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Wait;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class BstackdemoTest extends BaseTest {
+    @Test
+    public void addProductsToBasket() {
+        driver.get("https://www.bstackdemo.com/");
+        BstackdemoPage bstackdemoPage = new BstackdemoPage(driver);
+        String iphoneXsName = "iPhone XS";
+        WebElement buttonAddIphoneXs = bstackdemoPage.buttonAddProduct(iphoneXsName);
+        buttonAddIphoneXs.click();
+        String Pixel13Name = "Pixel 3";
+        WebElement buttonPixel3 = bstackdemoPage.buttonAddProduct(Pixel13Name);
+        buttonPixel3.click();
+        WebElement bagNameIphoneXs = driver.findElement(By.xpath("(//div[@class='float-cart__shelf-container']//p[@class='title'])[1]"));
+        WebElement bagPriceIphoneXs = driver.findElement(By.xpath("(//div[@class='float-cart__shelf-container']//div[@class='shelf-item__price'])[1]/p"));
+        WebElement bagQuantityIphoneXs = driver.findElement(By.xpath("(//div[@class='float-cart__shelf-container']//p[@class='desc'])[1]"));
+        WebElement bagNamePixel3 = driver.findElement(By.xpath("(//div[@class='float-cart__shelf-container']//p[@class='title'])[2]"));
+        WebElement bagPricePixel3 = driver.findElement(By.xpath("(//div[@class='float-cart__shelf-container']//div[@class='shelf-item__price'])[2]/p"));
+        WebElement bagQuantityPixel3 = driver.findElement(By.xpath("(//div[@class='float-cart__shelf-container']//p[@class='desc'])[2]"));
+        WebElement buttonCheckout = driver.findElement(By.xpath("//div[@class='buy-btn']"));
```

Comment:
```suggestion
        WebElement buttonCheckout = driver.findElement(By.className("buy-btn"));
```
Probably, can be simplified, the same for similar locators